src/index.js

```diff
@@ -23,17 +31,94 @@ export default class Proctor {
         recordViolation: true,
         ...config.tabSwitch,
       },
+      [VIOLATIONS.browserBlur]: {
+        name: VIOLATIONS.browserBlur,
+        enabled: true,
+        showAlert: true,
+        recordViolation: true,
+        ...config.browserBlur,
+      },
+      [VIOLATIONS.rightClick]: {
+        name: VIOLATIONS.rightClick,
+        enabled: true,
+        showAlert: true,
+        recordViolation: true,
+        ...config.rightClick,
+      },
+      [VIOLATIONS.exitTab]: {
+        name: VIOLATIONS.exitTab,
+        enabled: true,
+        showAlert: true,
+        recordViolation: true,
+        ...config.exitTab,
+      },
+      [VIOLATIONS.copyPasteCut]: {
+        name: VIOLATIONS.copyPasteCut,
+        enabled: true,
+        showAlert: true,
+        recordViolation: true,
+        ...config.copyPasteCut,
+      },
+      [VIOLATIONS.restrictedKeyEvent]: {
+        name: VIOLATIONS.restrictedKeyEvent,
+        enabled: true,
+        showAlert: true,
+        recordViolation: true,
+        ...config.restrictedKeyEvent,
+      },
+      [VIOLATIONS.textSelection]: {
+        name: VIOLATIONS.textSelection,
+        enabled: true,
+        showAlert: true,
+        recordViolation: true,
+        ...config.textSelection,
+      },
+      [VIOLATIONS.fullScreen]: {
```

Comment:
Also for fullscreen you have to add callbacks for enable fullscreen and disable fullscreen. if you take the latest pull you can see how different callbacks are integrated.